src/main/java/org/folio/rest/client/OkapiClient.java

```diff
@@ -93,9 +99,11 @@ public <T> Future<T> getById(String resourcePath, String id, Class<T> objectType
       int responseStatus = response.statusCode();
       log.debug("[{} {}ms] GET {}", responseStatus, currentTimeMillis() - start, resourcePath);
       if (responseStatus != 200) {
-        final String errorMessage = format("Failed to get %s by ID %s. Response status code: %s",
+        log.error("Failed to get {}} by ID {}}. Response status code: {}}",
```

Comment:
Can we also log response body?
```suggestion
        log.error("Failed to get {} by ID {}. Response status code: {}",
```